### ISSUE TITLE
Some questions about the losses

### ISSUE THREAD

**Issue by Vandermode** [ceciliavision/perceptual-reflection-removal#5]
Hi, I would like to know something about your losses setting. 
It seems the implementation of exclusion loss does not conform with the one described in the paper, e.g. the sigmoid function instead of tanh, which confuses me a little bit.

Also, I was wondering why not impose a L1/L2 loss to constraint the transmission layer. In my own experience, imposing such loss would enhance the performance in synthetic data, but degrade the performance in real data. So I would like to see your insight about this point.

**Comment by ceciliavision** [comment]
Hi, there might be a typo in the paper, thanks for pointing that out. Please use the code as reference (sigmoid), that's what we used for our experiments.

For your second question, we actually did. If you take a closer look at the function compute_percep_loss, there is p0 computed for 'input', which would be L1 between target and our output. It's embedded into the perceptual loss. We didn't do ablation on whether to have it or not (e.g. whether to have p0 or not), but it's an interesting experiment to try out.

**Comment by Vandermode** [comment]
Thank you for pointing out my mistake. So the L1 loss included in compute_percep_loss has already constrained the reflection layer in image pixel level, the additional L1 loss would only enhance the weight there. 

I still have some ablation-related questions, but due to the limited computation resources, I cannot evaluate all of them myself.

1. the hypercolumn feature is very expensive, how is the performance impact of such feature?  (ablation w/ or w/o hyper-column)

2. is the multi-resolution training strategy useful? (I will try myself, but was wondering your insight here)
(I notice this trick is borrowed from "Fast image processing", nevertheless, its impact is not mentioned.)

and I would like to know the training time of your model. 
Since the provided synthetic dataset size is larger than the one described in the paper (16000+ v.s. 5000), training such model is somewhat unacceptable for quick development. In my server, it costs about one day to run 10 epoch, which is too slow for me. 

==========UPDATE=========
By the way, It seems using this code cannot reproduce the real data result in pretrained model.
I guess it's mainly due to the usage of real data. So I was wondering how you generated 500 real images using the provided ones (90)
